Fix #8097: Re-fix markdown table rendering

kkthxbye-code opened the pull request:
### Fixes: #8097

The CSS for markdown-python table styling (https://github.com/netbox-community/netbox/commit/2fce7ebd8fee511a0fef8b3aada11037e8ef8e9f) was not ported to 3.0. This PR re-adds the CSS and adds the ´rendered-markdown´ CSS class where missing.

jeremystretch commented:
Rather than encapsulating every instance of `render_markdown` in an element with the `rendered-markdown` class, it may make more sense to just output `<div class="rendered-markdown">{content}</div>` from the filter directly. This would ensure it's not missed in future implementations of the filter.

kkthxbye-code commented:
Makes sense, I'll try to implement that.

kkthxbye-code committed:
Fix #8097: Re-fix markdown table rendering

kkthxbye-code commented:
render_markdown now wraps the output as you suggested. I added a check for empty strings as `render_markdown` is chained with `placeholder` in some spots. 

Removed any other usages of rendered-markdown.

jeremystretch committed:
Merge branch 'develop' into fix-8097
